.github/pull_request_template.md

```diff
@@ -34,8 +27,15 @@ https://kamma.atlassian.net/browse/GET-XXXX
 - [ ] Breaking change (fix or feature that would cause existing functionality to change)
 
 ## Checklist
+
+### Code style
 <!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
 <!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
 - [ ] My code follows the code style of this project.
 - [ ] My change requires a change to the documentation.
 - [ ] I have updated the documentation accordingly.
+
+
+### Security and validation
+- [ ] The code has appropriate validation
+- [ ] All raw database queries use prepared statements
```

Comment:
I like this idea! Not having a security section but two sections on detail does sound silly in retrospect haha

Could we add the following to that list as well?

```suggestion
- [ ] All raw database queries use prepared statements
- [ ] All external data, e.g. from API's, data sets or form POST, is validated and handled in its entirety and defensively
- [ ] Any vulnerability alerts for associated packages are patched
- [ ] The base Docker image, if it applies, is up to date with the latest OS image and packages
```